client/packages/common/src/ui/forms/JsonForms/components/Label.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {
+  LayoutProps,
+  rankWith,
+  UISchemaElement,
+  uiTypeIs,
+} from '@jsonforms/core';
+import { withJsonFormsLayoutProps } from '@jsonforms/react';
+import { SxProps, Typography } from '@mui/material';
+import { RegexUtils } from '@common/utils';
+
+export const labelTester = rankWith(3, uiTypeIs('Label'));
+
+type LabelVariant = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'p';
+
+type LayoutPropsExtended = LayoutProps & {
+  uischema: UISchemaElement & {
+    sx?: SxProps;
+    text?: string;
+    variant?: LabelVariant;
+  };
+};
+
+const variants: { [key in LabelVariant]: SxProps } = {
+  h1: { fontSize: '1.8em', fontWeight: 'bold', textAlign: 'center' },
```

Comment:
wider discussion: these should really come from the theme

Reply:
Yes, I'd like to standardise h1, h2, etc, but I don't want to go overwriting theme values for those just yet as it'll probably have significant flow-on effects. To do though: https://github.com/openmsupply/openmsupply-client/issues/1149